charts/argo-cd/values.yaml

```
@@ -266,6 +285,8 @@ configs:
     server.disable.auth: false
     # -- Enable GZIP compression
     server.enable.gzip: true
+    # -- Enable proxy extension feature
```

Can we mention the alpha phase here, please?

```suggestion
    # -- Enable proxy extension feature. (proxy extension is in Alpha phase)
```

---

@mkilchhofer 

Thank you for your feedback. The changes have been committed. Please kindly review.

---

@charlychiu thank you for implementing the propoal 👍